src/extensions/resolveSource.ts

```diff
@@ -23,7 +22,12 @@ export async function resolveSource(extensionName: string): Promise<string> {
   const extensionsRegistry = await getExtensionRegistry();
   const extension = _.get(extensionsRegistry, name);
   if (!extension) {
-    throw new FirebaseError(`Unable to find extension source named ${clc.bold(name)}.`);
+    throw new FirebaseError(
+      `Unable to find extension source named ${clc.bold(name)}. ` +
+        `Run ${clc.bold(
```

Comment:
We use resolve source in a bunch of places other than ext:install (for example, ext:update) - can you move this specific language out into the command?

(To be fair, since we call it with the extensionName from an existing instance in ext:update, this error shouldn't ever be thrown there, but i think its better to keep logs that are specific to a command inside that command)

Reply:
Good point, this now takes in a parameter which is a custom error message